### DISCUSSION TITLE
Future Ready Talent internship progress

### DISCUSSION THREAD

**Discussion by konduyx18** [github/india#490] [Students]
Hello everyone,
I am a new intern. Could you please help me to understand the structure. I already completed the orientation and started to study Azure. My question is that I assume I completed the section Describe Core Azure Concepts but my personal page says in progress I am not able to start the next lesson. Am I missing something? Thanks a lot for your time. Take care!

Best,
Yeliz

**Comment by No1-found** [comment]
It's same for all, don't stop there keep learning remaining modules
